src/main/java/seedu/address/logic/commands/FindContactCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.person.ContactContainsKeywordPredicate;
+
+/**
+ * Finds and lists all persons in address book whose name contains any of the argument keywords.
+ * Keyword matching is case-insensitive.
+ */
+public class FindContactCommand extends Command {
+
+    public static final String COMMAND_WORD = "findc";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds the person whose contact matches any of "
```

Comment:
"Finds the person whose **name** matches any of..."

We are using 2 separate commands, `findn` and `findp` so I think its necessary to specify that this search is based on the name of the contact/client.

Reply:
You may refer to the MVP Spec document

Reply:
I think it is right? I said finds there person whose contact matches to the keyword specified